src/backend/sev/snp/launch/mod.rs

```diff
@@ -18,37 +18,41 @@ use std::os::unix::io::AsRawFd;
 
 use bitflags::bitflags;
 
+use kvm_bindings::kvm_enc_region;
+
+use kvm_ioctls::VmFd;
```

Comment:
```suggestion
use bitflags::bitflags;
use kvm_bindings::kvm_enc_region;
use kvm_ioctls::VmFd;
```

Reply:
BTW, why there is empty line between use std::* and use::bitflags? Anyway the issue is sorted now. I'm not still exactly sure when to add a newline and when not.